docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -2008,7 +2049,7 @@ Mimir accepts timestamps that are slightly in the future, due to skewed clocks f
 On a per-tenant basis, you can fine tune the tolerance by configuring the `creation_grace_period` option.
 
 {{< admonition type="note" >}}
-Only series with invalid samples are skipped during the ingestion. Valid samples within the same request are still ingested.
+Invalid exemplars are skipped during ingestion. Valid exemplars in the same request are ingested.
```

Comment:
note to reviewers: this is not just style change, but also a bugfix. We check exemplars after samples are ingested, so the statement was wrong.